tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -52,3 +66,32 @@ def dismiss_tooltip(self):
         confirm_element = Wait(self.marionette).until(expected.element_present(*self._button_popup_intro))
         Wait(self.marionette).until(expected.element_displayed(confirm_element))
         confirm_element.tap()
+
+
+class BugzillaLiteStage(BugzillaLite):
+
+
+    name =  'Bugzilla Lite Stage'
+    manifest_url = 'http://bzlite-staging.herokuapp.com/manifest.webapp'
+
+
+    def dismiss_content(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._content_locator))))
+        self.marionette.find_element(*self._close_locator).tap()
+        Wait(self.marionette).until(expected.element_displayed(*self._login_form_locator))
+        import time
+        time.sleep(1)
+
+    def search(self, number):
+        searchbar_element = self.marionette.find_element(*self._search_bar)
+        searchbar_element.tap()
+        number_to_enter = self.keyboard.send(number)
+        searchbar_element.send_keys(number_to_enter)
+
+    def cancelSearch(self):
+        searchbar_element = self.marionette.find_element(*self._search_bar)
```

Comment:
I ran the test 3 times, and it was failing each time at this line. The element cannot be found. Once your python issue is fixed, can you tell me if it passes on your side?